src/rt-app.c

```diff
@@ -415,6 +434,72 @@ static int run_event(event_data_t *event, int dry_run,
 			pthread_yield();
 		}
 		break;
+	case rtapp_fork:
+		{
+			log_debug("fork");
```

Comment:
log of other events provide more information: log_debug("fork %s", rdata->res.fork.ref)

Reply:
Done